frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
 
 Our goal is to reach a point where every view in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
 
 As such, the design system is still in its infancy and will grow with each release.
 
+## Purpose (what it is and what it's not)
+
+This Storybook-based design system documentation only seeks to describe the actual reusable components; it does not seek to document features where these components are used.
+
+The purpose of this document is to:
+
+- Have a single point of truth on what reusable components should look like, how they should behave, and what options they provide.
+- Allow product designers and front-end developers to have a shared understanding of how these common elements are specified and implemented
```

Comment:
No, the full(!) specification might be too much detail for the DS. It is not what a product designer or developer needs. She needs to know what the intention behind a component is and how it should and should not get used. For the implementation we can look into the code. Having a full specification, one that suffices for implementation and QA testing is not a light task. It creates huge costs. Currently we don't have a central specification of OP, not even if you take all software tests together. I believe we should not lightly slip into the overhead in creating such a specification. 

```suggestion
- Allow product designers and front-end developers to have a shared understanding of how the intention behind these common elements and how they should and should not get used.
```

Reply:
I fail to see the core difference in your suggestion. Of course we will not be able to add a full specification of how a component might get used in feature X or Y. But we should strive for a complete picture on the component in isolation. This complete picture is derived from the specification and implementation rounds we have while spotifying a component. The result of it should absolutely make it into the DS

Reply:
I look at it from the perspective: What does a developer or product designer need to know to _reuse_ the component. That for sure should go into the DS here. However, I don't see a reason why we should repeat style definitions here in text form that are implemented in CSS if those style definitions are not needed for reusing the component/pattern. Everybody can look them up in the code or with the browser's dev tools when needed.